buildSrc/src/main/groovy/io/micronaut/guides/GuideAsciidocGenerator.groovy

```diff
@@ -252,4 +288,32 @@ class GuideAsciidocGenerator {
                 return ''
         }
     }
+
+    private static String resolveTestExtension(TestFramework testFramework) {
+        switch (testFramework) {
+            case JUNIT:
+                return 'java'
+            case SPOCK:
+                return 'groovy'
+            case KOTEST:
+            case KOTLINTEST:
+                return 'kt'
+            default:
+                return '@languageextension@'
+        }
+    }
+
+    private static String resolveLangFolder(TestFramework testFramework) {
```

Comment:
I think we already have methods for this in starter. see: `Language::getTestDir`

Reply:
Done!